homeassistant/components/reolink/binary_sensor.py

```diff
@@ -41,6 +45,18 @@ class ReolinkBinarySensorEntityDescription(
     value: Callable[[Host, int], bool]
 
 
+@dataclass(frozen=True, kw_only=True)
+class ReolinkSmartBinarySensorEntityDescription(
```

Comment:
```suggestion
class ReolinkSmartAIBinarySensorEntityDescription(
```
Since the feature is called "smart AI" and "smart" only is very generic, I would suggest using SmartAI here